core-services/prow/02_config/_config.yaml

```diff
@@ -49,6 +49,12 @@ branch-protection:
         rpm-ostree:
           protect: false
     cri-o:
+      repos:
+        cri-o:
+          exclude:
+          - master
```

Comment:
do we want to add main to future proof this?

Reply:
Makes sense, added `main`